work/worker.go

```diff
@@ -914,6 +910,31 @@ CommitTransactionLoop:
 		}
 	}
 
+	if atomic.LoadInt32(&abort) == 1 {
+		timeLimitReachedCounter.Inc(1)
+		var txHash common.Hash
+		if len(incorporatedTxs) > 0 {
+			txOrGen := incorporatedTxs[0]
+			tx, err := txOrGen.GetTx(env.state.GetNonce(nodeAddr))
+			if err == nil {
+				txHash = tx.Hash()
+			}
+			totalTxs := len(arrayTxs) + len(bundles)
+			logger.Warn("Unexecuted transactions due to time limit",
+				"txs", len(arrayTxs),
+				"bundlesTxs", len(bundles),
+				"totalTxs", totalTxs,
+				"executed", env.tcount,
+				"unexecuted", totalTxs-env.tcount,
+				"lastUnexecuted", txHash.String(),
+			)
+		}
+		if env.tcount == 1 && len(arrayTxs) > 0 {
+			logger.Error("A single transaction exceeds total time limit", "hash", arrayTxs[0].Hash().String())
```

Comment:
This might not be true when abortion happens during the execution of the second transaction. For example, `consumedByFirstTx = 200ms, consumedBySecondTx = 100ms`.

Reply:
I meant the log message `A single transaction exceeds total time limit`. When the second tx got caught by the time constraint, then `env.tcount == 1 && len(arrayTxs) > 0` but it's not by a **single** transaction.

Reply:
Ahh, okay, you're arguing the semantics of the message. First of all, I preserved the message as it is. Your scenario would work, but a heavy single tx scenario would also hit. 

I'll make a state variable that indicates `vm.ErrTotalTimeLimitReached` was met. If that variable is false, then the current message would work. Please add your input on the expected change.

Reply:
Fixed at  https://github.com/kaiachain/kaia/pull/639/commits/9174d7ac001b91ae724141086de63e8a1776b353